### PR TITLE
42 code example where tested with various SAST tools detecting flaws:

### PR DESCRIPTION
**Pull request by myteron**
- PyLint 6
- Bandit 3
- Pyre 1
- flake8 0
- Pysa 0

Some type hint related flaws are corrected as part of this PR

fixed wrong type hints in:
CWE-664/CWE-134/compliant01.py
CWE-664/CWE-843/compliant01.py
CWE-682/CWE-1335/01/compliant02.py
CWE-693/CWE-184/compliant01.py
CWE-693/CWE-184/noncompliant01.py

renamed CWE-664/CWE-502/compliant01.py to example as it does not represent compliance.

### CODE REVIEW

File: docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-665/example01.py
Comment (wbqpk3) [inline docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-665/example01.py:1]: I agree with the proposal to move this code into documentation.

File: docs/Secure-Coding-Guide-for-Python/CWE-682/CWE-1335/01/compliant02.py
Comment (wbqpk3) [inline docs/Secure-Coding-Guide-for-Python/CWE-682/CWE-1335/01/compliant02.py:1]: This correctly fixed the type annotation issue.

**Review by wbqpk3 — APPROVED** [review]
Hi, I reviewed the PR.

The changes LGTM.

**Review by BartyBoi1128 — APPROVED** [review]
Changes look fine! I approve them!

**Pull request merged**